.github/workflows/collector-builder.yml

```diff
@@ -11,11 +11,12 @@ on:
     outputs:
       collector-builder-tag:
         description: The builder tag used by the build
-        value: ${{ jobs.build-builder-image.outputs.collector-builder-tag || 'master' }}
+        value: ${{ jobs.build-builder-image.outputs.collector-builder-tag || '3.16.x-195-g8f32e71fad' }}
```

Comment:
What was this image exactly? And would it be dropped before merging?

Reply:
This was taken from mauro's original vanilla PR and will be dropped before merging.